tests/pytests/test_json.py

```diff
@@ -609,6 +609,17 @@ def testDemo(env):
     res =env.cmd('FT.AGGREGATE', 'airports', 'sfo', 'SORTBY', '1', '@iata', 'LOAD', '1', '$')
     env.assertEqual(toSortedFlatList(res), toSortedFlatList(expected_res))
 
+@skip(cluster=True, no_json=True, asan=True)
+def test_JSON_RDB_load_fail_without_JSON_module(env: Env):
+    env.expect('FT.CREATE', 'idx', 'ON', 'JSON', 'SCHEMA', '$.t', 'TEXT').ok()
```

Comment:
How can you create the index `ON JSON` when `no_json=True`
(it should fail with `Invalid rule type: JSON`)

Reply:
I skip the test if we don't have JSON at all. I first create an index with the module, and then reload the RDB after relaunching redis without the module. The test has to be skipped if the module is not available for the first step

Reply:
Consider to also create an index `on HASH`, in addition to the index `ON JSON`, so it would be successfully loaded before, and then see we clean it up as well.